common/package.json

```diff
@@ -10,17 +10,17 @@
     "lint:fix": "npm run lint:check -- --fix"
   },
   "dependencies": {
-    "core-js": "^3.32.2",
+    "core-js": "^3.37.0",
     "h5p-types": "^5.1.0",
     "h5p-utils": "^4.0.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
```

Comment:
Changes to react and react-dom are only meant for priming repos for React 19. No changes have been made to the runtime.